### ISSUE TITLE
Where is the Iteration and Loss output?

### ISSUE THREAD

**Issue by fengbinchen** [yahoo/CaffeOnSpark#115]
I've tried mnist example on AWS and I didn't see any output like this:
I0215 04:45:41.444522 26306 solver.cpp:237] Iteration 0, loss = 2.45106
I0215 04:45:41.444772 26306 solver.cpp:253]     Train net output #0: loss = 2.45106 (\* 1 = 2.45106 loss)
......

Is this normal? Where can I find the Iteration vs Loss information? I can find the model and feature output in hdfs and the job finished without any exception.

**Comment by GnosisYu** [comment]
You can open Spark web UI to check these

**Comment by junshi15** [comment]
It should be in your executor log.
